prow/plugins/trigger/pr.go

```diff
@@ -128,6 +136,10 @@ func trustedPullRequest(ghc githubClient, pr github.PullRequest) (bool, error) {
 }
 
 func buildAll(c client, pr github.PullRequest) error {
+
+	if err := c.GitHubClient.RemoveLabel(pr.Base.Repo.Owner.Login, pr.Base.Repo.Name, pr.Number, needsOkToTest); err != nil {
```

Comment:
I'm assuming it's not an error to remove a label that doesn't exist on the PR or in the repo. Have you checked that manually?